### ISSUE TITLE
Karma and Todos for Amit Ohayon

### ISSUE THREAD

**Issue by zvili** [TechnionYP5777/Bugquery#2]
_From @AmitOhayon on October 31, 2016 13:11_

- [x]  Use a picture, photo or distinct, familiar, personal avatar.
- [x] Present yourself by a not too rude **nick**  
- [x]  Describe yourself in the bio page
- [x]  State that you are (not) in search for a job
_Copied from original issue: SpartanRefactoring/Spartanizer#680_

**Comment by zvili** [comment]
_From @AmitOhayon on October 31, 2016 13:11_

@YossiGil: Done.

**Comment by zvili** [comment]
_From @tonylekhtman on October 31, 2016 17:2_

Awesome Page!

**Comment by yossigil** [comment]
Cleanup the blurb in the issue's body

**Comment by AmitOhayon** [comment]
@yossigil Done, if i understand what you meant

**Comment by yossigil** [comment]
:+1:

**Comment by yossigil** [comment]
Can you summarize here all issues you dealt with towards Sprint 1?

@AdiOmari

**Comment by AmitOhayon** [comment]
Since you only consider issues, the only ones I can mention are #3 and #7.
Much work was done behind the scenes about studying eclipse plugin development, and since we have no background it was difficult to make specific issues about it.

**Comment by AmitOhayon** [comment]
right now I'm dealing with #17

**Comment by AmitOhayon** [comment]
Since most of my work this week has been supporting others, I'm listing my contributions here:

1. Helping Tony with setting up db on new server & outside access
2. Spring ramp-up for Ziv and Roded, so they can work on adding exceptions to the main page
3. Updating & fixing tests to match new db access design
